client/src/app/core/core-services/action.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+
+import { HttpService } from './http.service';
+
+/**
+ * Currently supported actions
+ */
+export enum ActionType {
+    TOPIC_CREATE = 'topic.create',
+    TOPIC_DELETE = 'topic.delete',
+    TOPIC_UPDATE = 'topic.update',
+    AGENDA_ITEM_CREATE = 'agenda_item.create',
+    AGENDA_ITEM_UPDATE = 'agenda_item.update',
+    AGENDA_ITEM_DELETE = 'agenda_item.delete',
+    MOTION_UPDATE = 'motion.update',
+    MOTION_DELETE = 'motion.delete',
+    MOTION_SORT = 'motion.sort',
+    MOTION_UPDATE_METADATA = 'motion.update_metadata',
+    COMMITEE_CREATE = 'commitee.create',
+    MEETING_CREATE = 'meeting.create',
+    MEETING_UPDATE = 'meeting.update',
+    MEETING_DELETE = 'meeting.delete'
+}
+
+export interface RequestInfo<T> {
+    action: ActionType;
+    data: any;
+}
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ActionService {
+    private readonly ACTION_URL = '/system/action/handle_request';
+
+    private constructor(private http: HttpService) {}
+
+    public async sendRequest<T>(action: ActionType, data: any[]): Promise<T> {
```

Comment:
One little thing to discuss: In most cases `data: any` would be sufficient. We need this feature for bulk-updates. We should provide an extra function for this, so in the usual cases, the user of this service does not have to wonder why an array is needed.